peachjam/templates/peachjam/_contact_form.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 <section id="contact" class="row my-4">
   <div class="col-lg-6">
-    <h3 class="my-3">{% trans 'Contact Us' %}</h3>
+    <h3 class="my-3">{% trans 'Contact form' %}</h3>
```

Comment:
`Contact us` is better

Reply:
It describes the intention for the user, rather than the thing on the page.

Reply:
i've removed the heading, that section already has one
![image](https://github.com/user-attachments/assets/d9487345-8044-41e8-8de6-dbf70eb262aa)